releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -14,6 +14,130 @@
 
 [Source code][source_code_web] | [Package (Maven)][package_web]
 
+## 5.9.0-beta.1 - January 11, 2024
+
+### Features Added
+
+* Added support for loading Feature Variants from Azure App Configuration.
+
+## 5.8.0 - December 13, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.0.13, 3.1.0-3.1.6, 3.2.0-3.2.0.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4, 2023.0.0-2023.0.0.
+
+### Features Added
+
+* Added support for Azure App Configuration Snapshots. Snapshots can be loaded by setting `spring.cloud.azure.appconfiguration.stores[0].snapshot-name` to the snapshot name. Snapshots can be created using the Azure CLI or the Azure Portal. See [here](https://docs.microsoft.com/azure/azure-app-configuration/concept-snapshots) for more information on snapshots.
+* Added support for trimming prefixes from keys,the default value is the key-filter when key-filter is used. The configuration is `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefix` and `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefixes`. [#
+7598](https://github.com/Azure/azure-sdk-for-java/pull/37598)
+
+## 4.14.0 - December 14, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.18.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.8.
+
+### Features Added
+
+* Added support for Azure App Configuration Snapshots. Snapshots can be loaded by setting `spring.cloud.azure.appconfiguration.stores[0].snapshot-name` to the snapshot name. Snapshots can be created using the Azure CLI or the Azure Portal. See [here](https://docs.microsoft.com/azure/azure-app-configuration/concept-snapshots) for more information on snapshots.
+* Added support for trimming prefixes from keys,the default value is the key-filter when key-filter is used. The configuration is `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefix` and `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefixes`. [#
+7598](https://github.com/Azure/azure-sdk-for-java/pull/37598)
+
+## 5.7.0 - November 08, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.5.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4.
+* Now, Spring Boot 3.2.0-RC1 and Spring Cloud 2023.0.0-RC1 are compatible with this release.
+
+## 4.13.0 - November 08, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.17.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.8.
+
+## 5.6.0 - October 24, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.3.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4.
+
+## Bugs Fix
```

Comment:
We try to follow these names for the categories:
- Breaking Changes
- Enhancements
- Bug Fixes